.github/actions/perform-deployment/index.js

```diff
@@ -0,0 +1,123 @@
+// Copyright 2014-2021 Scalyr Inc.
```

Comment:
Some tests for this action would also be nice, but I know this may require quite a bit of mocking.

Reply:
Yeah, that js code is really hard to test.